packages/core/modules-sdk/src/loaders/utils/load-internal.ts

```diff
@@ -208,17 +208,24 @@ export async function loadModuleMigrations(
 async function importAllFromDir(path: string) {
   let filesToLoad: string[] = []
 
+  const excludedExtensionsRegexp = /(\.ts\.map|\.js\.map|\.d\.ts)/
```

Comment:
nit:
```suggestion
  const excludedExtensionsRegexp = /(\.ts\.map|\.js\.map|\.d\.ts)$/
```

Reply:
Ends with, ill commit it for extra safety🤘